tests/test_graph_api.py

```diff
@@ -684,6 +685,7 @@ def test_get_unicode_url():
     mock_request.return_value.status_code = 200
 
     response = graph.get('https://www.facebook.com/christophernewportuniversity‎')
+    del response['headers']
```

Comment:
Ok, just pushed something that "works" but I think it's still probably a cop-out.  I'm not sure what to expect in the response since the headers vary from request to request...